app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form as FormikForm } from "formik";
+import { Button, Pane } from "neetoui/v2";
+import { Input, Select } from "neetoui/v2/formik";
+import PropTypes from "prop-types";
+
+import {
+  INITIAL_CONTACT_VALUES,
+  VALIDATION_SCHEMA,
+  INITIAL_ROLE_VALUES,
+} from "./constants";
+
+const Form = ({ onClose, handleSubmit, initialContactValues }) => {
+  const populateSelectOptions = options =>
+    options.map(tag => ({
+      label: tag,
+      value: tag,
+    }));
+
+  return (
+    <Formik
+      initialValues={initialContactValues}
+      validationSchema={VALIDATION_SCHEMA}
+      onSubmit={handleSubmit}
+    >
+      {({ setFieldValue }) => (
+        <FormikForm className="w-full">
+          <Pane.Body className="space-y-6">
+            <div className="flex w-full space-x-4">
+              <Input
+                label="First Name"
+                name="firstName"
+                placeholder="Enter first name"
+                size="large"
+                required
```

Comment:
Do we need `required` here? and also to the fields below?

Reply:
In the Figma design, there were `*` at the end of every field so, I thought it was to put `required`. Should I just replace it with `*` in the label field and remove the `required`?